README.md

```diff
@@ -71,9 +71,9 @@ curl()
        HttpResponse response = responseSupplier.get();
        LOGGER.info("I log something after the call, status code is {}",
        response.getStatusLine().getStatusCode());
-       return response;}))
-                      .connectionManager(new PoolingHttpClientConnectionManager ())
-                      .placeHolders(asList("fr-FR", "text/html")).build())
+       return response;})).httpClientBuilder(HttpClients.custom()
+                      .setConnectionManager(new PoolingHttpClientConnectionManager ())
```

Comment:
I think I'll just upgrade to http client 5 instead

Reply:
> I think I'll just upgrade to http client 5 instead

If you have reviewed my refactored code and upgraded to HTTP client 5, you only need to focus on the implementation of AbstractClient, and there are no dependencies on any other series of code.